app/views/curriculum/_breadcrumb.html.erb

```diff
@@ -1,3 +1,4 @@
+<!-- Alter BreadcrumbComponent to make current_page_name an optional param -->
```

Comment:
If this comment is a TODO pls say so, and whether it is a blocker for going live.

Reply:
Its a "nice to have" todo. Ideally we could just use the new BreadcrumbComponent in here.